x-pack/plugins/cloud_security_posture/public/common/api/use_fetch_detection_rules_alerts_status.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { useQuery } from '@tanstack/react-query';
+import { GET_DETECTION_RULE_ALERTS_STATUS_PATH } from '../../../common/constants';
+import { DETECTION_ENGINE_ALERTS_KEY } from '../constants';
+
+interface AlertStatus {
+  acknowledged: number;
+  closed: number;
+  open: number;
+  total: number;
+}
+
+export const useFetchDetectionRulesAlertsStatus = (tags: string[]) => {
+  const { http } = useKibana().services;
+
+  if (!http) {
+    throw new Error('Kibana http service is not available');
+  }
+
+  return useQuery<AlertStatus, Error>([DETECTION_ENGINE_ALERTS_KEY, tags], () =>
+    http.get<AlertStatus>(GET_DETECTION_RULE_ALERTS_STATUS_PATH, {
+      version: '1',
```

Comment:
is this for serverless? we might need to use a shared constant to keep those versions aligned

Reply:
yes, nice suggestion, thanks